modules/local/chrom_sizes.nf

```diff
@@ -0,0 +1,28 @@
+process CHROM_SIZES {
+    label 'process_nompi'
+
+    conda (params.enable_conda ? "conda-forge::gawk=5.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gawk:5.1.0' :
+        'quay.io/biocontainers/gawk:5.1.0' }"
+
+    input:
+    path fai 
```

Comment:
Lets make this `tuple meta, fai`, to avoid future problems and then make the output  a tuple meta and `chrom_sizes`